infrastructure/template.yaml

```diff
@@ -3,9 +3,6 @@ Transform: "AWS::Serverless-2016-10-31"
 Description: "Digital Identity IPV CRI TOY API"
 
 Parameters:
-  CodeSigningEnabled:
-    Type: "String"
-    Default: "true"
```

Comment:
Why has the code signing config been changed?

Reply:
cfn-lint issues, you had 2 codesigning config lookups 1 of which wasnt used, so I changed it to the "official" way to do it